src/test/groovy/com/gradle/BaseInitScriptTest.groovy

```diff
@@ -49,15 +49,13 @@ abstract class BaseInitScriptTest extends Specification {
         dvPlugin(DEVELOCITY, DEVELOCITY_PLUGIN_VERSION),
         dvPlugin(DEVELOCITY, '3.17'),
 
-        dvPlugin(GRADLE_ENTERPRISE, DEVELOCITY_PLUGIN_VERSION, true),
```

Comment:
We should probably continue to test with the last released version of `com.gradle.enterprise`, which is '3.19.2'.